routes/users.py

```diff
@@ -141,4 +141,71 @@ def delete(uid):
         session.commit()
         lock.release()
         return result
-                          
\ No newline at end of file
+
+@app.route("/users/<int:user_id>/add_type")
```

Comment:
add_type takes user_id as a parameter --- do not assume that it will apply to user_id.